modules/ballerina-core/src/main/java/org/ballerinalang/util/BLangConstants.java

```diff
@@ -55,4 +55,10 @@
     public static final String BALLERINA_BUILTIN_PKG_PREFIX = "ballerina";
 
     public static final String USER_HOME = "user.home";
+    
+    // Zero value for string
+    public static final String STRING_ZERO_VALUE = null;
```

Comment:
Why do we call this "zero" value? .. it sounds like a zero length string .. shall we call it "STRING_NULL_VALUE"? ..